### PR TITLE
Update Sunder and Rainbow enchants to VI and III respectively, as they are their new highest tiers.

### PR DESCRIPTION
**Pull request by KTrain5169**

### CODE REVIEW

File: items/PIGS_FOOT.json [hunk 1]
```diff
@@ -28,4 +28,4 @@
       "result": "PIGS_FOOT"
     }
   ]
-}
```
Comment (jani270) [inline items/PIGS_FOOT.json:31]: ?

**Review by jani270 — CHANGES_REQUESTED** [review]
Please update the NBT of the max enchant book and remove the weird pig foot change

File: items/PIGS_FOOT.json (hunk 1 above)
Comment (KTrain5169) [reply items/PIGS_FOOT.json:31]: I'm not sure why the weird thing happened but I'll fix both issues

**Comment by KTrain5169** [comment]
Will open new PR that points to master instead

**Pull request closed without merging**